src/App.tsx

```diff
@@ -1,26 +1,22 @@
-import React from 'react';
+import React, {useState} from 'react';
 import logo from './logo.svg';
 import './App.css';
+import ModalConfirmCode from './components/ModalConfirmCode';
+import './components/ModalConfirmCode'
 
 function App() {
+  const [modalActive, setModalActive] = useState<boolean>(false);
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <button 
+        className='open-btn' 
+        onClick={() => setModalActive(true)}
+      >
+        Open modal
+      </button>
+      <ModalConfirmCode active={modalActive} setActive={setModalActive}/>
```

Comment:
the props also need to contain 'is' or 'has':

isActive,
setIsActive

Reply:
Done